### DISCUSSION TITLE
How can I create a torrent?

### DISCUSSION THREAD

**Discussion by josecelano** [transmission-remote-gui/transgui#1472] [Q&A]
I don't see any option to create torrents.

**Comment by lighterowl** [comment]
It's because there isn't one. Transmission doesn't expose any functionality related to creating torrents via RPC. A Transmission installation usually (but not always, as it's not an essential component if you just use the daemon) provides `transmission-create` but that'd be installed wherever your daemon is running instead of the GUI.
